hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/InodeTree.java

```diff
@@ -682,6 +748,32 @@ protected InodeTree(final Configuration config, final String viewName,
     }
   }
 
+  /**
```

Comment:
Why wouldn't you always sort the mount points? It seems like a useful invariant.

Reply:
Because sorting changes the existing implementation if nested mount point is disabled(current state) and is also not required with more overhead.

Reply:
i have changed this to always sort